packages/shared/src/components/auth/ReadingReminder.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { ReactElement, useState } from 'react';
+import { ClickableText } from '../buttons/ClickableText';
+import { Radio } from '../fields/Radio';
+import { Dropdown } from '../fields/Dropdown';
+import Alert, { AlertParagraph, AlertType } from '../widgets/Alert';
+import { Button, ButtonVariant } from '../buttons/Button';
+import { useEnableNotification } from '../../hooks/notifications';
+import { NotificationPromptSource } from '../../lib/analytics';
+import { usePersonalizedDigest } from '../../hooks';
+import { UserPersonalizedDigestType } from '../../graphql/users';
+import { Timezone } from '../widgets/Timezone';
+
+const ReadingReminderOptions = [
+  { label: '09:00 ☕️', value: '9' },
+  { label: '12:00 🥪', value: '12' },
+  { label: '17:00 🛋️', value: '17' },
+  { label: 'Custom️', value: 'custom' },
+];
+
+export const ReadingReminderTimes = [
+  { value: 1, label: '01:00' },
+  { value: 2, label: '02:00' },
+  { value: 3, label: '03:00' },
+  { value: 4, label: '04:00' },
+  { value: 5, label: '05:00' },
+  { value: 6, label: '06:00' },
+  { value: 7, label: '07:00' },
+  { value: 8, label: '08:00' },
+  { value: 9, label: '09:00' },
+  { value: 10, label: '10:00' },
+  { value: 11, label: '11:00' },
+  { value: 12, label: '12:00' },
+  { value: 13, label: '13:00' },
+  { value: 14, label: '14:00' },
+  { value: 15, label: '15:00' },
+  { value: 16, label: '16:00' },
+  { value: 17, label: '17:00' },
+  { value: 18, label: '18:00' },
+  { value: 19, label: '19:00' },
+  { value: 20, label: '20:00' },
+  { value: 21, label: '21:00' },
+  { value: 22, label: '22:00' },
+  { value: 23, label: '23:00' },
+  { value: 24, label: '24:00' },
+];
+const ReadingReminderTimeOptions = ReadingReminderTimes.map(
+  (item) => item.label,
+);
+
+interface ReadingReminderProps {
+  onClickNext: () => void;
+}
+export const ReadingReminder = ({
+  onClickNext,
+}: ReadingReminderProps): ReactElement => {
+  const [timeOption, setTimeOption] = useState('9');
+  const [customTimeIndex, setCustomTimeIndex] = useState(8);
```

Comment:
Making the first index of the `ReadingReminderTimes` with a value of `0` would make the JS management of time easier.

Reply:
You mean to set the first to 09:00? 
Feel it might become messier that way?

Reply:
Oh, sorry, I meant the `00:00` be the first item and having a value of `0`.